### ISSUE TITLE
Targets in Directory.Build.targets or .props not shown 

### ISSUE THREAD

**Issue by solvingj** [Serg046/MsBuildTaskExplorer#69]
If a custom target is defined in either of the following implicitly imported files, it does not appear in `MsBuildTaskExplorer`

- `Directory.Build.targets`
- `Directory.Build.props` 

If instead, I move the target definition to a separate `.targets` file and import that targets file in `Directory.Build.targets`, it is shown properly.

**Comment by Serg046** [comment]
Where are those files located? MsBuildTaskExplorer scans the solution folder only.

**Comment by solvingj** [comment]
They're outside the solution folder. That explains the issue.

**Comment by Serg046** [comment]
@solvingj if MsBuild itself understands your files in that folder, maybe it is worth scanning that (and similar ones) folder as well. Just let me know if you are interested in that and we can support it somehow.

**Comment by solvingj** [comment]
Thanks for taking an interest.  If you haven't read about them already, here's an overview of the files I was using:

https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019

It does probably make sense to support them if it's not too difficult.

**Comment by Serg046** [comment]
Thanks for the link, looks like we should go to the root level by level
```
c:\users\username\code\test\case1
c:\users\username\code\test
c:\users\username\code
c:\users\username
c:\users
c:\
```

**Comment by solvingj** [comment]
Yes that seems like one way you could approach it.  To be honest though, I'm surprised there's no internal project API for enumerating all loaded targets regardless of the props file they're in, and/or enumerating all props file's that have been loaded during evaluation phase. If you haven't looked for such a thing already, perhaps consider it.

**Comment by Serg046** [comment]
I totally agree, I even thought that we do something like that but based on what I saw, when I was checking, we just look for the files manually. Thank you for highlighting it, hope there is a way to do it like you said.

**Comment by solvingj** [comment]
thanks for all your hard work!
